CHANGELOG.md

```diff
@@ -1,34 +1,10 @@
 # Changelog
 
-## [7.1.0] - 28.10.2021
-
-## Added
-- Patterns section
-- Login Pattern
-- Forms Pattern
-- Usage with NPM - section to applicable components
-- ```<nav>``` tags around Breadcrumbs
-- Node package upgrade - Split code into various bundles which makes it possible to import specific components and color variables
-
+## [7.1.1] - 12.11.2021
 
 ## Changed
-- Landing page entry cards
-- Toast component and documentation page
-- Updated list of Products using DG
-
-## Removed
-- Disabled state on Links
-- Styling rule on ```<small>``` tag
-- Whitespace on Payex svg logo
-
-## Fixed
-- Alert boxes with no content, due to the last release, on Dialog page and datepicker page
-- Togglebox focus state bug
-- Issues with root-relative links when importing Stylesheet
-- Text-default color. Is now #512b2b
-- Weird looking top-border on Topbar
-- Bugs when nesting accordions.
-- Scrolling issues with Dialog
-
-
-
+- Toast documentation
+- Toast styling in mobile view
+- Form pattern styling
+- Getting-started to Get-started folders and paths
+- Path to Logos from designGuide/assets to img/logo
```

Comment:
I would vote on changing this to something like "Documentation image paths/sources"

Reply:
I second this ☝️